libs/ui/avatar/brain/src/lib/brn-avatar.component.ts

```diff
@@ -9,7 +9,7 @@ import { BrnAvatarImageDirective } from './image';
 	changeDetection: ChangeDetectionStrategy.OnPush,
 	encapsulation: ViewEncapsulation.None,
 	template: `
-		<ng-container *ngIf="image?.canShow(); else fallback">
```

Comment:
@ajitzero - as an additional improvement do you mind just going ahead and changing this if to the new syntax. No need to keep the ngIf import anymore

Reply:
Sure! Also, I had a question regarding the *.template files. Do I need overwrite them manually or are they generated?

Reply:
Switched to the new control flow syntax!

Reply:
> Sure! Also, I had a question regarding the *.template files. Do I need to overwrite them manually or are they generated?

So there is a generator tool which you can run I believe which will update it but I'm checking if we should be updating normally as part of PR or if they get updated at some other point. I'll get back to you

Reply:
Sure! I've updated the templates manually for all of my PRs as the changes were minor. I can revert that commit if it's being generated anyway.